bip-0044.mediawiki

```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
 
 ==Compatible wallets==
 
-* [[https://mytrezor.com|myTREZOR web wallet]] ([[https://github.com/trezor/webwallet|source]])
-* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 @ Android]] ([[https://github.com/ksedgwic/Wallet32|source]])
 * [[https://play.google.com/store/apps/details?id=com.mycelium.wallet|Mycelium Bitcoin Wallet (Android)]] ([[https://github.com/mycelium-com/wallet|source]])
 * [[https://copay.io/|Copay]] ([[https://github.com/bitpay/copay|source]])
+* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 (Android)]] ([[https://github.com/ksedgwic/Wallet32|source]])
 * [[https://maza.club/encompass|Encompass]] ([[https://github.com/mazaclub/encompass|source]])
 * [[https://www.coinvault.io/|CoinVault]] ([[https://github.com/CoinVault/dotblock|source]])
+* [[https://cancoin.co/|Cancoin]] ([[https://github.com/cancoin|source]])
+* [[https://samouraiwallet.com/|Samourai Wallet]] ([[https://github.com/Samourai-Wallet/samourai-wallet-android|source]])
+* [[https://www.darkwallet.is/|Darkwallet]] ([[https://github.com/darkwallet/darkwallet/|source]])
+
+* [[https://bitcointrezor.com/|TREZOR]] ([[https://github.com/trezor/|source]])
+* [[https://www.keepkey.com/|KeepKey]] ([[https://github.com/keepkey/|source]])
+* [[https://www.ledgerwallet.com/|Ledger Wallet]] ([[https://github.com/LedgerHQ|source]])
+* [[https://21.co/learn/21-lib-wallet/|21 Machine Wallet]] ([[https://github.com/21dotco|source]])
```

Comment:
@prusnak Might be even more of a reason to include it (BIPs are aimed at developers)